src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/Expander.cs

```diff
@@ -75,6 +71,11 @@ protected override void OnApplyTemplate()
 			}
 		}
 
+		// Uno specific: Added to dispose event handlers
+		_eventSubscriptions.Disposable = null;
+		var registrations = new CompositeDisposable();
+		_eventSubscriptions.Disposable = registrations;
+
```

Comment:
I think we should keep the subscription and unsubscription in Loaded/Unloaded

Reply:
The pattern of subscribing to things in OnApplyTemplate occurs throughout the codebase (see e.g. Slider). This shouldn't be leaking as long as we're only subscribing to child elements. I confirmed this with @MartinZikmund.

Reply:
Yes unless I am missing something, I think subscribing to a child should not leak 🤔 - do you see a problematic scenario there @Youssef1313 ?

Reply:
@MartinZikmund I'm not sure about iOS specifically where Swift GC can't handle circular references. https://learn.microsoft.com/en-us/xamarin/ios/deploy-test/performance#avoid-strong-circular-references I'm not sure if that's applicable here?

Reply:
I think one side of the circularity here is on the native side, and the other is managed, if I'm understanding correctly? Maybe it's not a problem in this case?

Reply:
Subscribing to a child will leak (through the delegate's target property) specifically on iOS. It's important to register/unregister to events in Load/Unload for that reason.

Reply:
We should also add that control the runtime leak tests (if not there already), to ensure we're not leaking:
https://github.com/unoplatform/uno/blob/93aa3e9825d6b3ee7d804649b40235569a3dd31a/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_And_Leak.cs#L130

Reply:
The odd thing is the control is already in the leak tests - so why does it not leak after this change according to CI 🤔 ?

Reply:
Ah! Maybe it could be because the test is not using fluent V2 styles!

Reply:
The fluent leak tests uncovered two leaks on Android, so already paying off. The PR should get merged soon, so then we can rebase this one and see whether the leak occurs.

Reply:
The leak tests were merged, can you rebase to see if it fails or not please?